src/contracts-api/Reader.ts

```diff
@@ -181,8 +197,18 @@ export default class Reader {
         id: res.id,
         token0: res.token0,
         token1: res.token1,
-        order0: res.order0,
-        order1: res.order1,
+        order0: {
```

Comment:
Why is it necessary?

Reply:
The object returning from the contract is a mix of array of values and an object - and if I simply use it as-is it eventually messes up with my serialization logic. I will tidy up the code.

Reply:
Roger that

Reply:
improved the code